test/runtests.jl

```diff
@@ -463,6 +463,15 @@ end
                             0.02118806218116165 0.6477623101170038
                             0.8543142404183606  2.564532433982956]
 
+    pred3 = predict(mm, newX, interval=:prediction)
+    @test pred1 == pred3[:, 1] ≈
+        [1.6488076594462182, 0.4706674451801356, 2.5010808086024423,
```

Comment:
I assume you checked this against another implementation?

Reply:
Yes, against R